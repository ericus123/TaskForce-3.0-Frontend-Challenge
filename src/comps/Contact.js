import { Row } from "react-bootstrap";
import "./styles.scss";
const Contact = () => {
    return (
        <div className="contact" id="contact">
            <div style={{color:"white", fontSize:"2em", textAlign:"center"}}>
                <h2><b>REACH ME</b></h2>
                <br/>
                <Row className="p-2.5">
                    
                    <h5 className="contact-h">Email</h5>
                    
                   
                    <h6 style={{color:"#ccc", fontWeight:"300"}}>amaniericus@gmail.com</h6>
                    
                </Row>
                <Row className="p-2">
                    
                    <h5  className="contact-h">Phone</h5>
                    
                   
                    <h6 style={{color:"#ccc", fontWeight:"300"}}>+(250)783343195</h6>
                    
                </Row>
                <Row className="p-2">
                    
                    <h5 className="contact-h">Profile</h5>
                    
                   
                    <h6><a style={{cursor:"pointer",color:"#ccc", fontWeight:"300"}} target="_blank" rel="noreferrer" href="https://rw.linkedin.com/in/amani-eric" className="text-decoration-none pointer">https://rw.linkedin.com/in/amani-eric</a></h6>
                    
                </Row>
            </div>
        </div>
    );
};
export default Contact;