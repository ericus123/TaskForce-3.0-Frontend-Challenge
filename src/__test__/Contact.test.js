/* eslint-disable no-undef */
import React from "react";
import { shallow,configure} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Contact from "../comps/Contact";
import toJson from "enzyme-to-json";

configure({adapter: new Adapter()});
it("renders contact without crashing", () => {
    let wrapper =  shallow(<Contact />);
    expect(toJson(wrapper)).toMatchSnapshot();
});

