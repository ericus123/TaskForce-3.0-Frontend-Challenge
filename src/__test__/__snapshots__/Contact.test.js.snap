// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders contact without crashing 1`] = `
<div
  className="contact"
  id="contact"
>
  <div
    style={
      Object {
        "color": "white",
        "fontSize": "2em",
        "textAlign": "center",
      }
    }
  >
    <h2>
      <b>
        REACH ME
      </b>
    </h2>
    <br />
    <Row
      className="p-2.5"
      noGutters={false}
    >
      <h5
        className="contact-h"
      >
        Email
      </h5>
      <h6
        style={
          Object {
            "color": "#ccc",
            "fontWeight": "300",
          }
        }
      >
        amaniericus@gmail.com
      </h6>
    </Row>
    <Row
      className="p-2"
      noGutters={false}
    >
      <h5
        className="contact-h"
      >
        Phone
      </h5>
      <h6
        style={
          Object {
            "color": "#ccc",
            "fontWeight": "300",
          }
        }
      >
        +(250)783343195
      </h6>
    </Row>
    <Row
      className="p-2"
      noGutters={false}
    >
      <h5
        className="contact-h"
      >
        Profile
      </h5>
      <h6>
        <a
          className="text-decoration-none pointer"
          href="https://rw.linkedin.com/in/amani-eric"
          rel="noreferrer"
          style={
            Object {
              "color": "#ccc",
              "cursor": "pointer",
              "fontWeight": "300",
            }
          }
          target="_blank"
        >
          https://rw.linkedin.com/in/amani-eric
        </a>
      </h6>
    </Row>
  </div>
</div>
`;
